### ISSUE TITLE
Cannot click if more than 6 menus

### ISSUE THREAD

**Issue by FitrahRama** [BCGDV-ASIA/android-widget-fanmenu#3]
I have 8 menus, but the two top of them cant be clicked.

**Comment by alphamu** [comment]
Can you paste your XML here.
I can imagine why this may be.
But it would help me to debug if I know how exactly you're using it.

**Comment by FitrahRama** [comment]
```
<?xml version="1.0" encoding="utf-8"?>

<android.support.v4.widget.DrawerLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/drawer_layout"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:openDrawer="start"
    tools:context=".views.activity.MainActivity">

    <android.support.design.widget.CoordinatorLayout
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:fitsSystemWindows="true">
        <android.support.design.widget.AppBarLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:theme="@style/AppTheme.AppBarOverlay">
            <ImageView
                android:id="@+id/ads"
                android:src="@drawable/sample_ads"
                android:background="#F0F0F0"
                android:scaleType="fitXY"
                android:padding="5dp"
                android:layout_width="match_parent"
                android:layout_height="75dp"/>
            <android.support.v7.widget.Toolbar
                android:id="@+id/main_toolbar"
                android:layout_width="match_parent"
                android:layout_height="?attr/actionBarSize"
                android:background="#c0ca33"
                android:gravity="center"
                app:popupTheme="@style/AppTheme.PopupOverlay" >
                <ImageView
                    android:src="@drawable/sehat_fresh"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:contentDescription="@string/app_name"/>
            </android.support.v7.widget.Toolbar>
        </android.support.design.widget.AppBarLayout>
        <android.support.v7.widget.CardView
            android:id="@+id/main_search"
            android:layout_width="match_parent"
            android:layout_height="50dp"
            app:layout_behavior="@string/appbar_scrolling_view_behavior">
            <LinearLayout
                android:orientation="horizontal"
                android:layout_marginRight="10dp"
                android:layout_marginLeft="10dp"
                android:layout_width="match_parent"
                android:layout_height="50dp">
                <EditText
                    android:maxLines="1"
                    android:singleLine="true"
                    android:id="@+id/main_search_edit"
                    android:layout_weight="0.2"
                    android:hint="Search"
                    android:layout_width="match_parent"
                    android:layout_height="50dp"
                    android:layout_marginLeft="5dp"
                    android:layout_marginRight="5dp"/>
                <ImageView
                    android:id="@+id/main_search_close"
                    android:src="@mipmap/ic_close"
                    android:layout_weight="0.8"
                    android:layout_width="match_parent"
                    android:layout_height="50dp"
                    android:layout_marginLeft="5dp"
                    android:layout_marginRight="5dp"/>
                <ImageView
                    android:id="@+id/main_search_done"
                    android:src="@mipmap/ic_done"
                    android:layout_weight="0.8"
                    android:layout_width="match_parent"
                    android:layout_height="50dp"
                    android:layout_marginLeft="5dp"
                    android:layout_marginRight="5dp"/>
            </LinearLayout>
        </android.support.v7.widget.CardView>
        <FrameLayout
            android:id="@+id/main_frame_layout"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            app:layout_behavior="@string/appbar_scrolling_view_behavior"/>
        <android.support.design.widget.FloatingActionButton
            android:id="@+id/fab"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_gravity="bottom|end"
            android:layout_margin="10dp"
            app:backgroundTint="@color/sehat_fresh"
            android:src="@drawable/sehatfresh_kotal" />
        <com.bcgdv.asia.lib.fanmenu.FanMenuButtons
            android:background="#CCFFFFFF"
            android:id="@+id/myFABSubmenu"
            android:layout_alignBaseline="@+id/myFAB"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:textColor="@android:color/white"
            android:paddingRight="@dimen/fab_margin"
            android:paddingBottom="@dimen/fab_margin"
            android:textSize="14sp"
            app:fanMenuLabels="@array/idesehat"
            app:fanButtonColors="@array/button_colors"
            app:fanRememberMovedPosition="true"
            app:fanMinBounceBackAngle="-30"
            app:fanMaxBounceBackAngle="50"
            android:visibility="gone"/>
    </android.support.design.widget.CoordinatorLayout>

    <android.support.design.widget.NavigationView
        android:id="@+id/nav_view"
        android:layout_width="wrap_content"
        android:layout_height="match_parent"
        android:layout_gravity="start">
        <ExpandableListView
            android:layout_marginTop="25dp"
            android:id="@+id/expandable_list"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"/>
    </android.support.design.widget.NavigationView>

</android.support.v4.widget.DrawerLayout>

```

**Comment by alphamu** [comment]
@FitrahRama the issue should be resolved in v1.2 by commit 5555fbaa8d8cd7e414e5f83fe318b367631e0096.
The problem was caused by a check that is used to make sure that no action happen if the user clicks too close to the base of the buttons as this action can be ambiguous. The check was only for length and not height of the click location. It should now be checking both.

If you do use the menu, please let us know, we would love to maintain a list of known apps using the widget.
